### PR TITLE
Fix parsing secret fields

### PR DESCRIPTION
**Pull request by jackcloudquery**

### CODE REVIEW

File: src/components/form/configUIForm.tsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ const FormStepper = React.lazy(() =>
  */
 export interface ConfigUIFormProps {
   prepareSubmitValues: (
+    config: PluginConfig,
```
Comment (cduflo) [inline src/components/form/configUIForm.tsx:42]: confirming this is the breaking change - yes?

**Review by cduflo — APPROVED** [review]
Great catch, probably saving us a big headache down the line 😀

File: src/components/form/configUIForm.tsx (hunk 1 above)
Comment (jackcloudquery) [reply src/components/form/configUIForm.tsx:42]: yes, I just don't see any other way to get config, so people must explicitly pass it in their `prepareSubmitValues` function from the arguments to `corePrepareSubmitValues`

Comment (cduflo) [reply src/components/form/configUIForm.tsx:42]: agreed on that 👍

**Pull request merged**